### ISSUE TITLE
More customers saying they cannot order

### ISSUE THREAD

**Issue by gregjotau** [Shopify/dawn#266]
**Describe the current behavior**

I have an iPhone 6 plus, am not 100 percent sure that safari is the latest but am 100 percent sure that chrome is. And I updated the actual phone yesterday as well. You can search for items but are not allowed to order them or see the overview of what kind of products you have or if they are sold out. Got to order via PC.
She I know has a Samsung A71 which is a newer type (another friend that  cannot order either)

**Describe the expected behavior**

**Version information (Dawn, browsers and operating systems)**
- Dawn Version: 1.0.0
- Chrome Version 91.0.4472.114
- macOS Version 11.3.1

**Possible solution**

**Additional context/screenshots**
Add any other context about the problem here. If applicable, add screenshots to help explain.

This is a customer saying they and their friend cannot order from their phone.

**Comment by gregjotau** [comment]
Unfortunately I do not have that, just forwarding feedback from a customer

**Comment by trenkwill** [comment]
Yes one of my clients is having issues specifically with Safari. Clicking Add to cart does nothing..
Here's a video that my client sent: https://www.youtube.com/watch?v=oBi4HpmQ3ac
It works perfectly in Firefox (first part of the video) but nothing happens in Safari. She seems to be on Safari 12.1.2

Thanks for the help!

**Comment by bredowmax** [comment]
I'm on Safari 14.1.1 and Safari 15.0 Preview on MacOS and I can't replicate. I noticed that @gregjotau you mentioned this happening on Dawn version 1.0 - does this still persist with the current version v1.1. of Dawn?

**Comment by martinamarien** [comment]
I've looked into this a bit more. I am able to replicate this when I simulate Dawn v1.1.0 on Safari 12.1. 

It looks like this issue is happening because we are using [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining), which is supported for Safari 13.1 and later. Since this is affecting people's ability to add items to their cart, I will create an issue to remove optional chaining for Dawn.
